client/src/store/auth-context.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export const AuthContext = React.createContext({
+  isLoggedIn: false,
```

Comment:
having username would be also useful to have if we needed to display that, could be email as well.